.gitignore

```diff
@@ -37,6 +37,7 @@ cmd/snap-mgmt/snap-mgmt
 cmd/snap-seccomp/snap-seccomp
 cmd/snap-update-ns/snap-update-ns
 cmd/snap-update-ns/unit-tests
+cmd/snapd-apparmor/snapd-apparmor
```

Comment:
Meh, produces a 5MB binary, or 4MB without symbol table and DWARF. Not that big, but the package will be ~70MB in side overall.

AFAICT the binary includes net (?), unicode (?), usersession, dbus, go-gettext, golang.org/x/text/unicode and as expected the import tree is rather large and full of surprises: https://paste.ubuntu.com/p/y7tQtxQScq/

Maybe we could reshuffle some of the internal imports, eg. apparmor.LoadProfiles move to interfaces/apparmor/internal? dirs.SnapAppArmorDir could be hardcoded. `strutil` probably shouldn't need to import yaml just for the sake of a strutil.OrderedMap (maybe yamlutil)? osutil imports randutil which pulls in crypto, so I'm suspecting some larger static tables.

Reply:
IMHO, it is okay to land this as-is with the large size of it and trim up the size of it later on, but I'm still rather skeptical that we can really get the size down much further, since Go binaries are just large.